include/electionguard/ballot.hpp

```diff
@@ -471,6 +542,55 @@ namespace electionguard
         std::unique_ptr<Impl> pimpl;
     };
 
+    /// <summary>
+    /// A CompactCiphertextBallot is a CompactPlaintextBallot that includes the encryption parameters
+    /// to properly re-encrypt the same ballot.
+    ///
+    /// This class is space optimized to serve specific use cases where an encrypted ballot is used
+    /// to verify that plaintext selections have not been tampered with.
+    ///
+    /// Don't make this directly. Use `make` instead.
+    /// </summary>
+    class EG_API CompactCiphertextBallot
```

Comment:
move this to another file like python so it can hold a reference to the manifest